plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -50,6 +50,12 @@
     private Duration serviceCacheTtl = new Duration(3, MINUTES);
     private boolean queryResultsCacheEnabled;
 
+    private int rpcInitialChannelCount = 1;
+    private int rpcMaxChannelCount = 2;
+    private int rpcMinChannelCount = 1;
```

Comment:
nit: The current order is rpcMax, rpcMin, minRpc, maxRpc. It would be better to replace rpcMax and rpcMin.